CHANGES.rst

```diff
@@ -1,3 +1,12 @@
+Unreleased
+==================
+
+- colander.All now supports Invalid.msg being None or a list, otherwise
```

Comment:
```suggestion
- :class:`colander.All` now supports :attr:`colander.Invalid.msg` being ``None`` or a list, otherwise
```

Reply:
It turns out that this suggestion is rejected when running lint checks. Please try this:

```
``colander.All`` now supports :attr:`colander.Invalid.msg` being ``None`` or a list, otherwise
```

Similarly, if `:attr:` and `:meth:` cause lint check failures, then remove those directives and double-backtick the attribute and method names.